README.md

```diff
@@ -5,26 +5,38 @@
 </div>
 <br/>
 
-# Table of Contents <!-- omit in toc -->
-
-- [Introduction](#introduction)
-- [Quickstart (Shannon)](#quickstart-shannon)
-- [Configuration](#configuration)
-  - [Configuration File](#configuration-file)
-    - [Example Configuration Format](#example-configuration-format)
-- [Running PATH](#running-path)
-  - [Setup Config YAML](#setup-config-yaml)
-  - [Start the Container](#start-the-container)
-- [E2E Tests](#e2e-tests)
-  - [Running Tests](#running-tests)
-
-## Introduction
+- [1. Introduction](#1-introduction)
+  - [1.1 Prerequisites](#11-prerequisites)
+- [2. Quickstart (Shannon)](#2-quickstart-shannon)
+- [3. Configuration](#3-configuration)
+  - [3.1 Configuration File](#31-configuration-file)
+  - [3.2 Example Configuration Format](#32-example-configuration-format)
+- [4. Running PATH](#4-running-path)
+  - [4.1 Setup Config YAML](#41-setup-config-yaml)
+  - [4.2 Start the Container](#42-start-the-container)
+- [5. E2E Tests](#5-e2e-tests)
+  - [5.1 Running Tests](#51-running-tests)
+- [6. User Data](#6-user-data)
+  - [6.1 Updated Endpoint](#61-updated-endpoint)
+  - [6.2 Database Configuration](#62-database-configuration)
+  - [6.3 Database Schema](#63-database-schema)
+
+## 1. Introduction
```

Comment:
> ULTRA NIT: My recommendation is to avoid numbers. I've fallen down this path myself.
> 
> Whenever you edit or move things around, you end up needing to re-number everything.

I have no strong preference in this regard but the Markdown plugin you suggested actually has a feature that auto-numbers everything based on the H1, H2, H3, etc. tags. that's where the numbers comes from. Happy to remove though; your call.

Reply:
As long as you're not spending manual time doing it, no objections.